### PR TITLE
Add Poppup and work section dynamicly

### PR DESCRIPTION
**Pull request by yash244466666**
Added the following files for Popup and Work Section.

- [work-cards.js](https://github.com/yash244466666/Microverse---Portfolio-setup-and-mobile-first/blob/popupjs/work-cards.js)
- [popup-app.js](https://github.com/yash244466666/Microverse---Portfolio-setup-and-mobile-first/blob/popupjs/popup-app.js)

### CODE REVIEW

File: work-cards.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        <h3 class="project-1-h3">Multi-Post Stories</h3>
+        <p class="project-1-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the .</p>
+    </div>
+    <div class="siklls">
+        <ul class="skills-ul">
+            <li><a href="${cardInfo[0].link}">css</a></li>
+            <li><a a href="${cardInfo[0].link}">html</a></li>
+            <li><a a href="${cardInfo[0].link}">bootstrap</a></li>
+            <li><a a href="${cardInfo[0].link}">Ruby</a></li>
+        </ul>
```
Comment (brytebee) [inline work-cards.js:22]: - [x] **Required:** Kindly update your styles to match the template design.

Your Design | Template
:---: | :---:
image | image

1. Kindly increase the size of the header or increase its weight.
2. Move the language tags to align to the left as the header above it and the image beneath.

- [x] **Optional:**

Your Design | Template
:---: | :---:
image | image

1. I personally think the buttons are too tall. Kindly reduce the margin/padding-top and bottom.
2. Also, reduce the size of the icons.
3. Increase the font size.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @yash244466666 👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yash244466666** [comment]
Add changes as instructed

- Fixed the Header
- Fixed the spacing

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @yash244466666 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - Good commit message.✔️
 - Good PR Summary.✔️
 - Great UI design.✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**